Exercises/NoModules/Chapter 03/Soln3_05.cpp

```diff
@@ -0,0 +1,45 @@
+// Exercise 3-6. Using an enumeration type for colors.
+// Of course, you have to research the RBG components for the colors.
+// Biggest complication is that you cannot apply any binary or other arithmetic operations
+// on values of a scoped enumaration type. For this, you have to first cast them to an integer.
+
+#include <iostream>
+#include <format>
+
+int main()
+{
+  enum class Color : unsigned
+  {
+    Red    = 0xFF0000u,
+    Green  = 0x00FF00u,
+    Blue   = 0x0000FFu,
+    Yellow = 0xFFFF00u,
+    Purple = 0xFF00FFu,
+    Black  = 0x000000u,
+    White  = 0xFFFFFFu
+  };
```

Comment:
The text says "Create variables of type Color initialized with enumerators for yellow, purple, and green"
I don't see these variables.

Reply:
Silly text...